scripts/ci/verify_linter.py

```diff
@@ -70,9 +70,17 @@ def _cmd(cmd):
         check_call(cmd, shell=True)
 
     def add_from_code(self):
+        # The containerapp-preview extension is a special case,
+        # it must depend on the continerapp extension and cannot run independently.
+        if self.extension_name == 'containerapp-preview':
+            self._cmd('azdev extension add containerapp')
         self._cmd('azdev extension add {}'.format(self.extension_name))
 
     def remove(self):
+        # The containerapp-preview extension is a special case,
+        # it must depend on the continerapp extension and cannot run independently.
+        if self.extension_name == 'containerapp-preview':
+            self._cmd('azdev extension remove containerapp')
```

Comment:
why do we need to remove containerapp when removing containerapp-preivew

Reply:
For pipeline linter test for containerapp-preview extension, need to install containerapp extension. 
So, need to remove containerapp when finish test for linter.